subprojects/docs/src/samples/userguide/basePlugin/kotlin/settings.gradle.kts

```diff
@@ -0,0 +1 @@
+rootProject.name = 'base-plugin-example'
```

Comment:
I think this should be double quotes `"`

Reply:
Of course. Good catch. I would have expected the integration tests to fail though, and they didn't. Did I miss something?

Reply:
Get it. I forgot to change the .conf file in order to execute both builds.